src/components/FlatParameters/FlatParameters.js

```diff
@@ -0,0 +1,265 @@
+            if (inEdit) {
+                setInEditionParam(param.name);
+            } else {
+                if (onChange && uncommitted != null) {
+                    if (['INTEGER', 'DOUBLE'].includes(param.type)) {
+                        onChange(param.name, uncommitted - 0, false);
+                    } else {
+                        onChange(param.name, uncommitted, false);
+                    }
+                }
+                setInEditionParam(null);
+                setUncommitted(null);
+            }
+        },
+        [uncommitted, onChange]
+    );
+
+    const renderField = (param) => {
+        const value =
+            param.name === inEditionParam && uncommitted !== null
+                ? uncommitted
+                : initValues && initValues.hasOwnProperty(param.name)
+                ? initValues[param.name]
+                : extractDefault(param);
+        switch (param.type) {
+            case 'BOOLEAN':
+                return (
+                    <Switch
+                        checked={!!value}
+                        onChange={(e) => onFieldChange(e.target.checked, param)}
+                    />
+                );
+            case 'DOUBLE':
+                return (
+                    <TextField
+                        fullWidth
+                        value={value}
+                        onFocus={() => onUncommited(param, true)}
+                        onBlur={() => onUncommited(param, false)}
+                        onChange={(e) => {
+                            const m = FloatRE.exec(e.target.value);
+                            console.debug(
+                                'double.onChange',
+                                value,
+                                m,
+                                e.target.value
+                            );
+                            if (m) {
+                                onFieldChange(e.target.value, param);
+                            }
+                        }}
+                        variant={'standard'}
+                    />
+                );
+            case 'INTEGER':
+                return (
+                    <TextField
+                        fullWidth
+                        value={value}
+                        onFocus={() => onUncommited(param, true)}
+                        onBlur={() => onUncommited(param, false)}
+                        onChange={(e) => {
+                            const m = IntegerRE.exec(e.target.value);
+                            console.debug(
+                                'double.onChange',
+                                value,
+                                m,
+                                e.target.value
+                            );
+                            if (m) {
+                                onFieldChange(e.target.value, param);
+                            }
+                        }}
+                        variant={'standard'}
```

Comment:
why overwriting the default variant of the TextField ? Should be parametrized for all the FlatParameters component maybe ?

Reply:
It used to be that way in useImportExport, may be in order not to take too much space

Reply:
If used in multiple places it has an impact, like in LF parameters in GridStudy, today fields are of outlined variant.
Maybe juste add a global `variant` prop in FlatParameters component to be used for every fields of parameters

Reply:
done